schemas/isd.json

```diff
@@ -0,0 +1,242 @@
+                "key": {"type": "string"},
+                "focal_length": {"type": "number"},
+                "focal_epsilon":{"type": "number"}
+            },
+   	"required": ["focal_length"]
+       },
+       "IFOV": {
+           "description": "Instantaneous field of view in degrees",
+           "type": "number"
+       },
+       "IMAGE_LINES": {
+           "description": "The number of lines in the image",
+           "type": "number"
+       },
+       "IMAGE_SAMPLES": {
+           "description": "The the number of samples in the image",
+           "type": "number"
+       },
+
+       "RADII": {
+           "description": "The radii of the target.",
+           "type": "array",
+	    "items": {
+                 "type": "number",
+		 "minItems": 1,
+                 "maxItems": 3
+            }
+       },
+       "OPTICAL_DISTORTION": {
+           "description": "Information needed to convert from undistorted to distorted image plane",
+           "type": "object",
+	   "properties" : {
+	      "coefficients": {
+	         "type": "array",
+		 "items": {
+		    "type": "number"
+		  }
+		},
+	        "key" : {
+		    "type": "string",
+		    "description": "key into LUT to determine equation to use"
+		    }
+	 },
+	"required": ["coefficients", "equation_key"]
+       },
+
+       "STARTING_DETECTOR_SAMPLE": {
+           "description": "The sample on the detector where the data starts",
+           "type": "number"
+       },
+       "STARTING_DETECTOR_LINE": {
+           "description": "The line on the detector where the data starts",
+           "type": "number"
+       },
+       "FOCAL2PIXEL_X": {
+           "description": "Transformation x coefficients from focal plane (mm) to pixel coordinates",
+           "type": "array",
+	          "items": {
+                 "type": "number"
+            }
+       },
+       "FOCAL2PIXEL_Y": {
+           "description": "Transformation y coefficients from focal plane (mm) to pixel coordinates",
+           "type": "array",
+	          "items": {
+                 "type": "number"
+            }
+       },
+       "SENSOR_LOCATION": {
+           "description": "(X, Y, Z) location of sensor in body-fixed frame.",
+           "type": "array",
+	          "items": {
+               "type": "array",
+	              "items" : {
+                  "type": "number",
+                  "minItems": 3,
+                  "maxItems": 3
+		              }
+	               }
+      },
+       "SENSOR_VELOCITY" : {
+           "description": "(v_x, v_y, v_z) velocity of sensor in body-fixed frame.",
+           "type": "array",
+	          "items": {
+               "type": "array",
+	              "items" : {
+                    "type": "number",
+                    "minItems": 3,
+                    "maxItems": 3
+		                }
+	                 }
+	     },
+       "SUN_POSITION" : {
+           "description": "(v_x, v_y, v_z) velocity of sun in body-fixed frame.",
```

Comment:
This description is a duplicate of the velocity description.

Reply:
:+1:

Reply:
:+1: